### PR TITLE
[NS-72] Rename plugin configuration options

### PR DESCRIPTION
**Pull request by q10**
Summary:

- Rename all Spark Cyclone plugin configuration options away from the `com.nec` namespace.  Configuration options will no longer be prefixed with `spark.com.nec` but with `spark.cyclone`
- Move `spark.cyclone.spark.ncc` config options to `spark.cyclone.ncc`
- Rename `spark.cyclone.spark.kernel.directory` to `spark.cyclone.kernel.directory`
- Fix documentations and scripts that include outdated configuration spellings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,25 +101,25 @@ This will try to pre-shuffle the data so that we only need to call the VE in one
...
---conf spark.com.nec.spark.project-on-ve=true
---conf spark.com.nec.spark.filter-on-ve=true
---conf spark.com.nec.spark.exchange-on-ve=true
---conf spark.com.nec.spark.pass-through-project=false
---conf spark.com.nec.spark.fail-fast=false
---conf spark.com.nec.spark.join-on-ve=false
---conf spark.com.nec.spark.amplify-batches=true
---conf spark.com.nec.spark.ve.columnBatchSize=<spark col batch size>
---conf spark.com.nec.spark.ve.targetBatchSizeMb=64
+--conf spark.cyclone.spark.aggregate-on-ve=true
```
Comment (wmeddie) [inline README.md:112]: Possible to rename these options to spark.cyclone.sql.xxx-on-ve?

**Review by wmeddie — CHANGES_REQUESTED** [review]
Looks good but I think we should take the chance to rename the xxx-on-ve options as well.

File: README.md (hunk 1 above)
Comment (q10) [reply README.md:112]: Sure

Comment (q10) [reply README.md]: Done

**Review by wmeddie — APPROVED** [review]

**Pull request merged**